src/html.rs

```diff
@@ -108,13 +113,13 @@ where
                     self.write_newline()?;
                 }
                 HardBreak => {
-                    self.write("<br />\n")?;
+                    self.write(&(String::from("<br />") + BACKLINE))?;
```

Comment:
Are these allocating? If so, that seems like a performance hit.

Reply:
It's const string cancatenation so it should have no impact at runtime (to be confirmed though!).

Reply:
Ah maybe not. I think `String::from` isn't const.

Reply:
Originally I wanted to use `if cfg!(feature = "no-backline")` but found it too noisy. Well in any case, it's another possibility. But the question here is really if you're interested by this feature or not. If you are, I need to add tests too. :)

Reply:
Possibly. But considering the small impact, I think it's not worth it currently. Maybe for later on.